src/test/java/org/openstreetmap/atlas/checks/event/CheckFlagGeoJsonProcessorTestRule.java

```diff
@@ -44,7 +44,8 @@
 
                     @Area(coordinates = { @Loc(value = TEST_5), @Loc(value = TEST_2),
                             @Loc(value = TEST_4), @Loc(value = TEST_1),
-                            @Loc(value = TEST_6) }, tags = { "building=yes" }) })
+                            @Loc(value = TEST_6) }, tags = { "building=yes",
+                                    "highway=notanenumvalue" }) })
```

Comment:
is the function above called on areas?

Reply:
It is called on the jsonified tags of all check flag objects. So yes, but not as an Area object.

Reply:
This should catch the solved bug, but why is it in a Area (a polygon)? Shouldn't you add it to:

```
edges = {

                    @Edge(coordinates = { @Loc(value = TEST_1), @Loc(value = TEST_2),
                            @Loc(value = TEST_3) }, tags = { "highway=primary" }) },
```

Reply:
It doesn't really matter too much which one the bad tag is added to. However, I wanted to keep the test realistic. This issue was detected when running the `AreasWithHighwayTagCheck`, and it was because the tag was not in an edge that it threw the exception. Normally, something with a highway tag is only ingested into an atlas if it has a highway tag that is in the Atlas enum set. This is due to the way the edge filter works. When something does get through as a non-edge with a highway tag, its inclusion in the enum is not guaranteed.

Reply:
Ah, thank you for the clarification. Makes sense!